### ISSUE TITLE
Add automated accessibility testing

### ISSUE THREAD

**Issue by erikkroes** [ing-bank/lion#293]
Automated testing can't catch all issues, but it would be a great improvement for the quality of the product if automated testing would be included.

The easiest way to do this would be to use chai-a11y-axe: https://www.npmjs.com/package/chai-a11y-axe
It's part of the open-wc tooling that we already use.

**Comment by lucasjs** [comment]
@erikkroes  What about add storybook a11y addon (to check some violations, like contrast)?
https://github.com/storybookjs/storybook/tree/master/addons/a11y
I could create a pull request. :)

**Comment by erikkroes** [comment]
@lucasjs We would like that very much! We even have an open issue for that: #298 
They both have their uses, so having both would be great.
